src/main/interface/cli.c

```diff
@@ -2828,7 +2838,7 @@ STATIC_UNIT_TESTED void cliGet(char *cmdline)
     int matchedCommands = 0;
 
     for (uint32_t i = 0; i < valueTableEntryCount; i++) {
-        if (strstr(valueTable[i].name, cmdline)) {
+        if (strstr(valueTable[i].name, cliStringToLowercase(cmdline))) {
```

Comment:
`strcasestr` is cleaner alternative ...

Reply:
I started that way originally and `strcasestr` is not in ANSI C.  The examples I found to include were encumbered by BSD licensing so the recommendation was to the lowercase approach.  Then tried to use `strlwr` - again not part of ANSI.  Lastly write my own.

Reply:
Problem with `cliStringToLowercase` is that in unnecessarily modifies input argument ...

Reply:
Yes, but all implementations I could find of `strlwr` work this way.  The alternative would require allocating another buffer to store the result which seems messy for this limited case.  Also then trying to move to a "generic" standard function would be challenging.

Reply:
Implementing `strcasecmp` would do ...

Reply:
we allready have strcasecmp implemented https://github.com/betaflight/betaflight/blob/master/src/main/common/string_light.c#L59

and it is allready used in other places in this cli.c file ;)

Reply:
`strcasecmp` isn't needed in this context, `strcasestr` is.

Reply:
then we just should add it to the string_light.c and done :)

Reply:
Just need to rewrite it enough to not be bound by the BSD license it's currently released though.

Reply:
Also `strtol` is currently pulling in locale stuff, string_light.c is actually wasting flash ...